src/plugins/embeddable/public/lib/panel/embeddable_panel.tsx

```diff
@@ -200,10 +204,15 @@ export class EmbeddablePanel extends React.Component<Props, State> {
   };
 
   private getActionContextMenuPanel = async () => {
-    const actions = await this.props.getActions(CONTEXT_MENU_TRIGGER, {
+    let actions = await this.props.getActions(CONTEXT_MENU_TRIGGER, {
       embeddable: this.props.embeddable,
     });
 
+    const { disabledActions } = this.props.embeddable.getInput();
+    if (disabledActions) {
+      actions = actions.filter(action => disabledActions.indexOf(action.id) === -1);
```

Comment:
chore: disabledActions.includes(action.id) reads nicer to me

Reply:
I agree, `.includes()` is nicer, but `.indexOf()` is more widely supported (and I'm oldschool).

Reply:
Don't we have all needed polyfills in place?

Reply:
no idea